src/components/Attachment/AttachmentFileInfo.tsx

```diff
@@ -60,11 +60,15 @@ export const AttachmentFileInfo = forwardRef<AttachmentFileInfoProps, "div">(
         )}
         <Flex sx={styles.fileInfo}>
           <Icon as={BxImage} />
-          <Text ml="2rem">{file.name}</Text>
+          <Text ml="2rem" textStyle="caption-2" noOfLines={1}>
```

Comment:
nit: should add `word-break: break-all` else we get:

<img width="640" alt="Screenshot 2023-11-08 at 15 19 18" src="https://github.com/isomerpages/isomercms-frontend/assets/27919917/85d52387-988a-4c52-b0c5-1845e9cfed0f">

Reply:
can but my pathological case on `break-word` led to it overflowing to the side. i'll change it!

Reply:
swapped to enforcing `w=100%`, which prevented this case